src/vercel.js

```diff
@@ -36,7 +36,7 @@ const init = () => {
 			const metadata = [
 				`githubCommitAuthorName=${ commit.authorName }`,
 				`githubCommitAuthorLogin=${ commit.authorLogin }`,
-				`githubCommitMessage=${ commit.commitMessage }`,
+				`githubCommitMessage=${ commit.commitMessage.substring(0, 50) }`,
```

Comment:
Why did you choose to limit the commit message to exactly 50 characters? The 4kb limit should allow for a fair amount of text. As you mentioned, the first line of a commit message is often used as a subject, but it is not fixed in length (50 is just a convention).

Why not select the message up to the first line break? That way we can limit the message to the "subject line" however long it might be. Something like `commitMessage.split(/\r?\n/)[0]` should work.

Reply:
Awesome suggestion! The `50` came from a conventional commit pithy subject, but that should not be imposed here. Will update. 👍🏻